config/deploy.rb

```diff
@@ -45,3 +45,19 @@
     end
   end
 end
+namespace :sidekiq do
+  task :quiet do
+    on roles(:app) do
+      # Horrible hack to get PID without having to use terrible PID files
+      puts capture("kill -USR1 $(sudo initctl status plum-workers | grep /running | awk '{print $NF}') || :")
```

Comment:
Yuck. :ok_hand: Having to remember to set that up when we deploy to a new box is going to be a pain